modules/user_accounts/js/permissions.js

```diff
@@ -0,0 +1,375 @@
+'use strict';
+
+var Role = React.createClass({
```

Comment:
Thanks for your tips! I have been trying to figure out how we are doing React stuff this days; do you know if we documentation somewhere I can follow? I found this but out of date I guess: https://github.com/aces/Loris/blob/master/LORIS_react.README.md